tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -423,6 +423,22 @@ describe(__filename, () => {
     expect(link).toHaveProp('href', `/addon/${addon.slug}/versions/beta`);
   });
 
+  it('does not link to beta versions if they are disabled', () => {
+    const fakeConfig = new Map();
+    fakeConfig.set('betaVersions', false);
+
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      slug: 'some-slug',
+      current_beta_version: {
+        ...fakeAddon.current_version,
+        version: '3.0.0-beta',
+      },
+    },
+    { _config: fakeConfig });
```

Comment:
I know we don't have great lint rules for this but this should be indented to indicate that it's a continuation of the call up above. I'd suggest something more like:

```js
const addon = createInternalAddon(
  {
    ...fakeAddon,
    slug: 'some-slug',
  },
  { _config: fakeConfig },
);
```